src/plugins/foxPage/components/OtherOpportunities/OtherOpportunities.tsx

```diff
@@ -19,30 +19,28 @@ export const OtherOpportunities: React.FC<OtherOpportunitiesProps> = ({
 }) => {
   return (
     <Card display='block' width='full' borderRadius={8}>
-      <Card.Header pb={0}>
+      <Card.Header pb={0} mb={4}>
         <Flex flexDirection='row' alignItems='center' mb={2}>
           <Text translation={title} fontWeight='bold' color='inherit' />
         </Flex>
         <Text translation={description} color='gray.500' />
       </Card.Header>
-      <Card.Body pb={0}>
-        <Accordion mx={-6} defaultIndex={[0]} allowToggle allowMultiple>
-          {opportunities.map(opportunitiesBucket => {
-            const { opportunities } = opportunitiesBucket
-            if (!opportunities.length || opportunities.every(opportunity => opportunity.isDisabled))
-              return null
+      <Accordion defaultIndex={[0]} allowToggle allowMultiple>
+        {opportunities.map(opportunitiesBucket => {
+          const { opportunities } = opportunitiesBucket
+          if (!opportunities.length || opportunities.every(opportunity => opportunity.isDisabled))
+            return null
 
-            return (
-              <FoxOtherOpportunityPanel
-                key={opportunitiesBucket.type}
-                opportunities={opportunities}
-                title={opportunitiesBucket.title}
-                type={opportunitiesBucket.type}
-              />
-            )
-          })}
-        </Accordion>
-      </Card.Body>
+          return (
+            <FoxOtherOpportunityPanel
+              key={opportunitiesBucket.type}
+              opportunities={opportunities}
+              title={opportunitiesBucket.title}
+              type={opportunitiesBucket.type}
+            />
+          )
+        })}
```

Comment:
Not a blocker since this is just shuffling existing code around, but can we memoize the `opportunities.map()` expression?